### PR TITLE
Enable and apply linters for yaml and markdown 

### PR DESCRIPTION
**Pull request by pichfl**
Prettier supports both and consistency is always a sensible thing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,22 @@
-ember-promise-modals
-==============================================================================
+# ember-promise-modals
```
Comment (Turbo87) [inline README.md:1]: I would prefer to keep the old style for the headers since it looks better in plain ASCII form

Comment (pichfl) [reply README.md:1]: I could do something like this, but I really think that would be beside the point of using Prettier in the first place.

```markdown
<!-- prettier-ignore-start -->

ember-promise-modals
==============================================================================

<!-- prettier-ignore-end -->
```

Comment (Turbo87) [reply README.md:1]: yeah, that makes it look even worse.

Comment (pichfl) [reply README.md:1]: I can also ignore that file, but that also somewhat defeats the purpose

Comment (Turbo87) [reply README.md:1]: I assume we can't configure prettier to use the header style that we're already using here?

Comment (pichfl) [reply README.md:1]: There is no configuration

**Comment by pichfl** [comment]
Closing this for good.

**Pull request closed without merging**